spec/unit/providers/gce_spec.rb

```diff
@@ -52,39 +51,49 @@
 
   subject { Vmpooler::PoolManager::Provider::Gce.new(config, logger, metrics, redis_connection_pool, 'gce', provider_options) }
 
+  before(:each) { allow(subject).to receive(:dns).and_return(MockDNS.new()) }
+
   describe '#name' do
     it 'should be gce' do
       expect(subject.name).to eq('gce')
     end
   end
 
   describe '#manual tests live' do
-    skip 'runs in gce' do
-      puts 'creating'
-      result = subject.create_vm(poolname, vmname)
-      subject.get_vm(poolname, vmname)
-      subject.vms_in_pool(poolname)
-
-      puts 'create snapshot w/ one disk'
-      result = subject.create_snapshot(poolname, vmname, 'sams')
-      puts 'create disk'
-      result = subject.create_disk(poolname, vmname, 10)
-      puts 'create snapshot w/ 2 disks'
-      result = subject.create_snapshot(poolname, vmname, 'sams2')
-      puts 'revert snapshot'
-      result = subject.revert_snapshot(poolname, vmname, 'sams')
-      result = subject.destroy_vm(poolname, vmname)
-    end
-
-    skip 'runs existing' do
-      # result = subject.create_snapshot(poolname, vmname, "sams")
-      # result = subject.revert_snapshot(poolname, vmname, "sams")
-      # puts subject.get_vm(poolname, vmname)
-      result = subject.destroy_vm(poolname, vmname)
-    end
-
-    skip 'debug' do
-      puts subject.purge_unconfigured_resources(['foo', '', 'blah'])
+    context 'in itsysops' do
+      let(:vmname) { "instance-15" }
+      let(:project) { 'vmpooler-test' }
+      let(:config) { YAML.load(<<-EOT
```

Comment:
fwiw, if you change this to `<<~EOT` you can indent the yaml two spaces under the let block instead of having to outdent it to the edge.

Reply:
ok will do 👍